web/src/engine/osk/src/banner/bannerView.ts

```diff
@@ -165,129 +170,71 @@ export default class BannerView implements OSKViewComponent {
 }
 
 export class BannerController {
-  private _activeType: BannerType;
-  private _options: BannerOptions = {};
   private container: BannerView;
-  private alwaysShow: boolean;
-  private imagePath?: string = "";
 
   private predictionContext?: PredictionContext;
 
   private readonly hostDevice: DeviceSpec;
 
-  public static readonly DEFAULT_OPTIONS: BannerOptions = {
-    alwaysShow: false,
-    imagePath: ""
-  }
+  private _inactiveBanner: Banner;
```

Comment:
Is this the opposite of `activeBanner`? The terms active and inactive here seem really ambiguous

Reply:
... yeah, I bumped up against that for a bit.  Kind of makes me want to change the `.activeBanner` to `.currentBanner`.

It might be clearer if I also split `BannerView` and `BannerController` into separate files in the predecessor PR; `.activeBanner` is only in the former, while `.inactiveBanner` is only in the latter.

Of course, there's also the timeless classic question of "... why not both?"

Reply:
As of the newly-pushed commits, I've implemented both thoughts from my comment above.